internal/smbcc/container_config.go

```diff
@@ -99,6 +99,9 @@ const (
 	Yes = "yes"
 	// No means no.
 	No = "no"
+
+	// AddVFSFileid is used to chec if we add vfs_fileid to the config
+	AddVFSFileid = true
```

Comment:
This doesn't seem like the right approach to me, why is this a global rather than say, an argument to NewGlobals?

Reply:
I've added a new commit which changes the way we pass the configuration options. Let me know if this is what you had in mind. Not squashing patches until they are acceptable.